src/bridge/cockpitfsdir.c

```diff
@@ -300,6 +322,7 @@ cockpit_fsdir_class_init (CockpitFsdirClass *klass)
  * @transport: the transport to send/receive messages on
  * @channel_id: the channel id
  * @path: the path name of the file to read
+ * @nowatch: boolean, skip watching?
```

Comment:
This leads to double negatives like "nowatch: false".  I think "watch: true" and "watch: false" is nicer.